op-proposer/proposer/service.go

```diff
@@ -103,6 +107,9 @@ func (ps *ProposerService) initFromCLIConfig(ctx context.Context, version string
 	if err := ps.initL2ooAddress(cfg); err != nil {
 		return fmt.Errorf("failed to init L2ooAddress: %w", err)
 	}
+	if err := ps.initDGF(cfg); err != nil {
+		return fmt.Errorf("failed to init DGF: %w", err)
+	}
```

Comment:
Need to skip attempting to init these if the relevant address is the empty string.  Otherwise you get:
```
        	Error:      	Expected nil, but got: &fmt.wrapError{msg:"unable to setup l2 output submitter: failed to init DGF: invalid address: ", err:(*errors.joinError)(0xc0071132a8)}
```

As all the e2e tests are currently showing. :)

Reply:
Now I see why it used to return `nil` 🤣 

thanks!